### ISSUE TITLE
Constant Ticking Entity Crashes on server when in twilight forest labyrinths

### ISSUE THREAD

**Issue by Darkmega18** [TeamTwilight/twilightforest#565]
Forge version: 13.23.4.2705
Twilight Forest version: twilightforest-1.12.2-3.7.424-universal
Link to crash log: https://pastebin.com/N5NunpbM (and potentially like 12 more, just like that one at different times)

Steps to reproduce:
1. Not sure. But friend reports that he was in the twilight forest and he was exploring around in the labyrinths, and had apparently fought a naga and then a lich before going there. But while he was there the server was crashed a grand total of 13 times, all with the same crash reports at different times over the hour and a half it took him to finish it without crashing and waiting for a roughly 3-4 min startup time.

What I expected to happen:
sometimes these preset questions are dumb for certain things. but typically I'd have thought he wouldn't crash. :V 

What happened instead:
crash every few minutes, or as friend put it, "crash every 30 seconds". :<

**Comment by Andromander** [comment]
That's... a very strange crash log. The entity being ticked is a Cave Spider, and it's reported that it has a passenger Cave Spider and a vehicle Cave Spider, so I'm to assume there is an infinite stack of Cave Spiders trying to spawn for some reason.

It's likely this isn't on Twilight Forest's side, but I would suggest removing mods until the crashing stops, particularly mods that would deal with a Cave Spider.

**Comment by bs2609** [comment]
For whatever reason, something's set the spider to ride itself:
```
Entity Type: minecraft:cave_spider (net.minecraft.entity.monster.EntityCaveSpider)
Entity ID: 1987
Entity Name: Cave Spider
Entity's Exact location: -528.50, 9.00, 270.50
Entity's Block location: World: (-529,9,270), Chunk: (at 15,0,14 in -34,16; contains blocks -544,0,256 to -529,255,271), Region: (-2,0; contains chunks -64,0 to -33,31, blocks -1024,0,0 to -513,255,511)
Entity's Momentum: 0.00, -0.23, 0.00
Entity's Passengers: [EntityCaveSpider['Cave Spider'/1987, l='DimensionDivers', x=-528.50, y=9.00, z=270.50]]
Entity's Vehicle: EntityCaveSpider['Cave Spider'/1987, l='DimensionDivers', x=-528.50, y=9.00, z=270.50]
```

See that the ID of the ticking spider, the passenger spider and the vehicle spider is the same (1987).

**Comment by Darkmega18** [comment]
oh. well shit, i know exactly whats causing that then (*cough* rough mobs 2 *cough*). and something possibly related to that was fixed, but not sure about that direct issue, so I'll report it there. my bad for pointing it here. since I wasn't there when the friend had these shenanigans.

actually reminds me of that time in one of my old modpacks when I had the old homerun bats mod, and I made zombies that could have riding bats, and called them riding assassins. they's sneak up onto you and just whack you over and over and ride you into the clouds and you'd never be heard from again... but one time I managed to ride bat the ride assassin so it infinite stacked us until it crashed, and when I relogged I was like 10k blocks into the air. xD
